R/zchunk_data_Maddison_population.R

```diff
@@ -35,7 +35,7 @@ module_data_Maddison_population <- function(command, ...) {
       add_units("people") %>%
       add_comments(paste("Read from", gsub("^.*extdata", "extdata", fqfn))) %>%
       add_legacy_name("Maddison_population") %>%
-      add_flags(FLAG_LONG_YEAR_FORM, FLAG_NO_XYEAR, FLAG_NO_TEST) ->
+      add_flags(FLAG_LONG_YEAR_FORM, FLAG_NO_XYEAR, FLAG_NO_OUTPUT) ->
       Maddison_population
```

Comment:
I realized that Maddison_population not only doesn't need to be tested (it's not a data system output), but doesn't even need to be written to disk. This saves a few microseconds.